app/code/Magento/InventoryConfigurationAdminUi/etc/module.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryConfigurationAdminUi" setup_version="1.0.0"/>
```

Comment:
We don't need to use the `setup_version` since we are using declarative schema definition